.gitignore

```diff
@@ -1,9 +1,11 @@
 build/
 *.class
 *.crt
+.DS_Store
```

Comment:
I've excluded the project folder from Spotlight indexing... but still, Spotlight probably managed to create these files right before I did so